src/ui/GenioWindow.cpp

```diff
@@ -54,6 +54,7 @@
 #include "GoToLineWindow.h"
 #include "GSettings.h"
 #include "IconMenuItem.h"
+#include "json.hpp"
```

Comment:
this include is huge.. do we really need it? It looks like the change is only based on strings

Reply:
done